### PR TITLE
2016 Steering Committee application: Leanne Wake

### PR DESCRIPTION
**Pull request by lmwake**
Hi-
Hopefully you can access my application for the steering committee. I'm not great at GitHub, but I am learning and was determined not to submit this application via email!
Best,
Leanne

### CODE REVIEW

File: LEANNE_WAKE_STEERING2016.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+**Parallelisation:** Making your code more efficient. You have built a car - how do you turn it into a Ferrari?  How many folks do you know have a fancy multi-core machine, yet only use a single core? SWC is contributing towards reproducible science, how about efficient science?
+
+* *Organisational Duties/Miscellaneous:*
+
+I am willing to accept any admin duties to ensure fluidity and expansion of the organisation, e.g.:
+
+-**Maintaining a live instructor database** indicating temporal availability of instructors. 
+
+-**Recruitment:** Workshop and participants number are increasing; Instructor numbers are [tailing off.] (http://software-carpentry.org/about/)
```
Comment (wking) [inline LEANNE_WAKE_STEERING2016.md:45]: A stable instructor count should be able to maintain a linear learner increase ;).  But yeah, the more, the merrier :).

**Comment by jduckles** [comment]
@lmwake, thanks for the submission! You just need to add metadata for Jekyll at the top of the file, including the three `---` to open and close the metadata block:

```
 ---
 layout: post
 author: "Leanne Wake"
 title: "2016 Election: Leanne Wake"
 date: 2016-01-12
 time: "15:00:00"
 category: ["Software Carpentry Foundation"]
 ---
```

**Comment by lmwake** [comment]
Hi- Thanks for this. You'll receive the updated version soon.
Leanne

File: LEANNE_WAKE_STEERING2016.md (hunk 1 above)
Comment (lmwake) [inline LEANNE_WAKE_STEERING2016.md:45]: @wking  You do have a point! I guess I'd like to promote higher numbers of instructors partly due to the huge benefits SWC instructor training has had on my non-computational teaching techniques at my uni.
Leanne

**Comment by lmwake** [comment]
@jduckles The file is now updated (commit 7b4e3b5).
Cheers,
Leanne.

**Comment by jduckles** [comment]
@lmwake one more thing I missed earlier, this has to be put in `_posts/2016/01` and named `YYYY-MM-DD-shortname.md` or it won't be picked up by Jekyll. Thanks!

**Comment by lmwake** [comment]
I will close this pull request and open another.
Thanks
Leanne

**Pull request closed without merging**